src/static/css/homepage.css

```diff
@@ -0,0 +1,26 @@
+.asd {
+    padding: 60px;
+    text-align: center;
+    background: #6C7A89;
+    color: white;
+}  
+
+.asd > .title {
+    font-size: 50px;
+}
+.asd > .caption {
+    font-size: 20px;
+}
+
+.social-icons-container{
+    position:absolute;
+    top:0;
+    right:0; 
+    padding: 20px 10px;
+}
+
+.social-icons-container img{
+    width: 30px;
```

Comment:
do you need to set a width for this container? the div should scale automatically

Reply:
This is just to restrict the size of the image/icon for each img within the `social-icons-container`